docs/download-upload-clone/high-level-design.md

```diff
@@ -0,0 +1,56 @@
+# High Level Design
+
+* exporting a project exports its underlying scenarios
+* as the client agreed, pre-defined resources do not need to be described 
+  with unique identifier shared across possible instances of MarxanCloud 
+  (Databases) - if any shared resource with imported ID is not available, 
```

Comment:
this is a bit tricky.
in the current implementation, we cannot rely on stable db-generated UUIDs as proxies unique ids of datasets that are imported platform-wide via ETL pipelines, because the UUIDs would different between instances, yet the data may be exactly the same (for example, same release of GADM data from upstream).

we would need to add and handle some basic metadata such as unique dataset name/id (can be set when creating new ETL pipelines if these are included in the core repo), and version (this could also be part of the unique id, since we won't need to care if source is using GADM dataset vN and target is using GADM dataset vM - we'll need to consider them as two fully different datasets)

Reply:
actually, we may want to keep track of version separately, in case it may be ok to allow importing data where the same ETL-imported dataset is present as in the source instance, but in a later version. unlikely, but probably worth using an extra field for this.